app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
```

Comment:
1. by default it split by space, so you can simplify this
2. Before accessing values it would be good to check that len of command_list is exactly 3
3. check that first command is cp

Reply:
fixed